pkg/operator/csi/csidrivercontrollerservicecontroller/csi_driver_controller_service_controller.go

```diff
@@ -152,8 +151,8 @@ func (c *CSIDriverControllerServiceController) sync(ctx context.Context, syncCon
 	}
 
 	var clusterID string
-	if c.optionalConfigInformer != nil {
-		infra, err := c.optionalConfigInformer.Config().V1().Infrastructures().Lister().Get(infraConfigName)
+	if c.configInformer != nil {
```

Comment:
the name no longer says optional and you appear to require during construction.  Can this actually be nil?